Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[workspace]
+members = ["components/rust", "profiling"]
+
+[profile.dev]
+debug = 2 # full debug info
+
+[profile.release]
+debug = 1 # line tables only
+lto = true
+codegen-units = 1
+opt-level = "s" # optimize for size
```

Comment:
```
% cargo build
warning: profiles for the non root package will be ignored, specify profiles at the workspace root:
package:   %s/profiling/Cargo.toml
workspace: %s/Cargo.toml
```

So sadly, yes, this will change profiling from the default of `opt-level = 3` to `opt-level = "s"`.

I suppose for this PR we could delete the root `Cargo.toml` in the profiler build jobs? Any better ideas?

Reply:
I would be fine with adding a separate profile for the tracer release, i.e. `cargo build --profile tracer-release`?

Reply:
That seems to work fine, let me do that.